src/com/cli/contact/model/Contact.java

```diff
@@ -0,0 +1,21 @@
+package com.cli.contact.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Contact {
```

Comment:
@anghelitalo The class name indicates that is the representation of a single Contact, but you are handle a list of contacts, the name should be ContactManager or something like that